### ISSUE TITLE
Will you or did you try Group Normalization

### ISSUE THREAD

**Issue by XCRobert** [KimSoybean/ScratchDet#5]
According to your paper, BN plays a critical role training from scratch. Did you try Group Normalization ? Does Group Normalization have better perfomance than some pretrained ones.
According your assumption,it may also works.
![image](https://user-images.githubusercontent.com/16361120/55685248-73838680-5986-11e9-9740-8ae158c7c2a1.png)

**Comment by KimSoybean** [comment]
@XCRobert  Hello, I have tried GN on ScratchDet on mmdetection and get 1 mAP lower result than BN. Sorry I haven't found the reason but I think it is relevant to learning rate. But more details should be given in analysis with gradients. I will do that after I get my IELTS testing passed. Thank you!
